### PR TITLE
Chore comment validation

### PR DESCRIPTION
**Pull request by neymarjimoh**
**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**
- THis handles the schema validation for comment flag
Fixes #issue_number (replace issue_number with the issue number this PR solves)

Summarise the main tasks handled in the pull request

**description of Task to be completed?**

Describe (in detail) what the task completed in the pull request does as per the relevant task

**How should this be manually tested?**

Give bullet point instructions on how to setup the code and test it manually (Assume no prior experience on JS or code)

**Any background context you want to provide?**

Any pertinent information that should be considered

**What is the link to the issue on Github?**

[Issue #14](https://github.com/microapidev/comment-microapi/issues/14)

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (piouson) [inline LICENSE:1]: This PR removes the repo LICENSE. Please commit again and include the license.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -6,8 +6,6 @@
 
 **What does this PR do?**
 
-Fixes #issue_number (replace issue_number with the issue number this PR solves)
```
Comment (piouson) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: This PR alters a repo template. Please commit again with template unchanged.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
 # Comments Microservice
```
Comment (piouson) [inline README.md:1]: This PR alters the repo README. Please commit again and include an updated README.

File: __test__/api/comment-delete.test.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ const CommentModel = require("../../models/comments");
 const ReplyModel = require("../../models/replies");
 const mongoose = require("mongoose");
 const request = supertest(app);
-import { describeIfEndpoint } from "../helpers/conditionalTests";
```
Comment (piouson) [inline __test__/api/comment-delete.test.js:7]: you are submitting an outdated file

File: __test__/api/comment-get.test.js [hunk 5]
```diff
@@ -4,16 +4,12 @@ const CommentModel = require("../../models/comments");
 const mongoose = require("mongoose");
 const supertest = require("supertest");
 const request = supertest(app);
-const { describeIfEndpoint } = require("../helpers/conditionalTests");
```
Comment (piouson) [inline __test__/api/comment-get.test.js:7]: you are submitting an outdated file

File: __test__/api/comment-post.test.js [hunk 6]
```diff
@@ -3,9 +3,10 @@ const supertest = require("supertest");
 const CommentModel = require("../../models/comments");
 const mongoose = require("mongoose");
 const request = supertest(app);
-import { describeIfEndpoint } from "../helpers/conditionalTests";
```
Comment (piouson) [inline __test__/api/comment-post.test.js:6]: you are submitting an outdated file

**Comment by neymarjimoh** [comment]
Yeah, I'd have to update and commit again @piouson .

**Pull request closed without merging**